packages/react-native/bugsnag-react-native-xcode.sh

```diff
@@ -23,7 +23,14 @@ if [ ! -f "$BUNDLE_FILE" ]; then
   exit 0
 fi
 
-MAP_FILE="$BUNDLE_FILE.map"
+# Generate a stable unique path that does not contain spaces, for compatibility with react-native-xcode.sh
+UID=$(md5 -qs "$CONFIGURATION_BUILD_DIR")
+MAP_FILE="$TMPDIR/$UID-main.jsbundle.map"
+if [ ! -f "$MAP_FILE" ]; then
+  echo "Warning: $MAP_FILE could not be found, falling back to $BUNDLE_FILE.map"
+  # Older versions of the Xcode build phase used this location
+  MAP_FILE="$BUNDLE_FILE.map"
+fi
```

Comment:
Can we instead take this as a parameter so that the same value can be passed in as was supplied to `react-native-xcode.sh`? That way, if the user is setting this up manually they can set the directory to whatever they like.

Reply:
The build phase now looks like

<img width="834" alt="Screenshot 2021-02-16 at 11 30 06" src="https://user-images.githubusercontent.com/61777/108057285-591e9780-704a-11eb-84e6-e89dda660b26.png">